x-pack/filebeat/input/azureblobstorage/job.go

```diff
@@ -186,16 +189,21 @@ func (j *job) readJsonAndPublish(ctx context.Context, r io.Reader, id string) er
 		// updates the offset after reading the file
 		// this avoids duplicates for the last read when resuming operation
 		offset = dec.InputOffset()
+		var (
+			done func()
+			cp   *Checkpoint
```

Comment:
completely inconsequential nit that can be ignored; change to reflect assignment and use order.
```suggestion
			cp   *Checkpoint
			done func()
```